test/feature.go

```diff
@@ -177,47 +173,13 @@ func matchesTags(filter string, tags []string) (ok bool) {
 	return
 }
 
-func hasTag(tags []string, tag string) bool {
+func hasTag(tags []*messages.Pickle_PickleTag, tag string) bool {
```

Comment:
For consistency with the above methods, should we make the tags argument the last parameter?

```go
func hasTag(tag string, tags []*messages.Pickle_PickleTag) bool {
```